From 027dbc9f85f2e55af283d6714b0b1fed478900dd Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 20 Jan 2012 12:02:15 +0100
Subject: [PATCH] dmsquash-live: really changed /dev/live-baseloop to
 /run/initramfs/

Forgot to amend my change to ce32e32f2a8288de15968c22f57fff306fa81753
---
 modules.d/90dmsquash-live/dmsquash-live-root |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/90dmsquash-live/dmsquash-live-root b/modules.d/90dmsquash-live/dmsquash-live-root
index cb104e4..d9fcc43 100755
--- a/modules.d/90dmsquash-live/dmsquash-live-root
+++ b/modules.d/90dmsquash-live/dmsquash-live-root
@@ -201,7 +201,7 @@ if [ -n "$ROOTFLAGS" ]; then
 fi
 
 if [ -b "$BASE_LOOPDEV" ]; then
-    ln -s $BASE_LOOPDEV /dev/live-baseloop
+    ln -s $BASE_LOOPDEV /run/initramfs/live-baseloop
 fi
 ln -s /dev/mapper/live-rw /dev/root
 printf '/bin/mount %s /dev/mapper/live-rw %s\n' "$ROOTFLAGS" "$NEWROOT" > $hookdir/mount/01-$$-live.sh
